sources/packages/web/src/components/generic/StatusChip.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-badge :color="backGroundColor" :text-color="textColor" :class="badgeClass">
+    <template v-slot:badge>
+      <font-awesome-icon
+        :icon="['fas', 'circle']"
+        class="mr-1"
+        :color="iconColor"
+      />
+      <span class="text-uppercase">{{ label ?? status }}</span>
+    </template>
+  </v-badge>
+</template>
+<script lang="ts">
+import { computed } from "vue";
+import { StatusShipTypes } from "@/components/generic/StatusChip.models";
```

Comment:
Ship? did u mean chip?

Reply:
It is supposed to be StatusChipTypes. I will rename it.

Reply:
Fixed!